lib/matplotlib/legend.py

```diff
@@ -1096,6 +1103,43 @@ def _find_best_position(self, width, height, renderer, consider=None):
     def contains(self, event):
         return self.legendPatch.contains(event)
 
+    def set_draggable(self, state, use_blit=False, update='loc'):
+        """
+        Enable or disable mouse dragging support of the legend.
+
+        Parameters
+        ----------
+        state : bool
+            ``True`` / ``False`` enables / disables mouse dragging.
```

Comment:
I tend to write these as (e.g.) "Whether mouse dragging is enabled."

Reply:
That‘s a good formulation. Unfortunately this is already merged and I don‘t think it’s worth an extra PR. However, I will use it in the future.